src/Plugin/os2web/DataLookup/DatafordelerCVR.php

```diff
@@ -90,7 +94,7 @@ public function lookup($cvr) {
         $cvrResult->setFloor($address->CVRAdresse_etagebetegnelse ?? '');
         $cvrResult->setApartmentNr($address->CVRAdresse_doerbetegnelse ?? '');
         $cvrResult->setPostalCode($address->CVRAdresse_postnummer ?? '');
-        $city = $address->CVRAdresse_postdistrikt ?? '' . $cvrResult->getPostalCode();
+        $city = ($address->CVRAdresse_postdistrikt ?? '') . $cvrResult->getPostalCode() ?? '';
```

Comment:
I have a feeling that we're missing a space separator here …

```suggestion
        $city = implode(' ', array_filter([
          $address->CVRAdresse_postdistrikt ?? NULL,
          $cvrResult->getPostalCode() ?? NULL,
        ]);
```